### PR TITLE
feat(sdk,app): make kms storage injectable.

### PR DESCRIPTION
**Pull request by vkubiv**

### CODE REVIEW

File: cmd/wallet-sdk-js/docs/usage.md [hunk 1]
```diff
@@ -1,3 +1,148 @@
...
-TODO
\ No newline at end of file
+Last updated: Jul 18, 2023
+
+This guide explains how to use this SDK in Web. The examples in this document demonstrate
+how to use the various APIs from a JavaScript perspective.
+
+### Examples
+
+This example shows full opendid4ci flow.
```
Comment (DRK3) [inline cmd/wallet-sdk-js/docs/usage.md:10]:
```suggestion
This example shows the full OpenID4CI flow.
```

File: cmd/wallet-sdk-js/docs/usage.md [hunk 2]
```diff
@@ -1,3 +1,148 @@
...
+
+let parsedDisplayData = await agent.parseResolvedDisplayData({
+    resolvedCredentialDisplayData: rawDisplayData
+})
+
+```
+
+### KMS Database
+
+kmsDatabase should implement next 3 methods:
```
Comment (DRK3) [inline cmd/wallet-sdk-js/docs/usage.md:56]:
```suggestion
kmsDatabase should implement the next 3 methods:
```

**Review by DRK3 — APPROVED** [review]
Looks good! Some very minor comments

File: cmd/wallet-sdk-js/docs/usage.md [hunk 3]
```diff
@@ -1,3 +1,148 @@
...
+
+This guide explains how to use this SDK in Web. The examples in this document demonstrate
+how to use the various APIs from a JavaScript perspective.
+
+### Examples
+
+This example shows full opendid4ci flow.
+
+```javascript
+let pathWhereWalletsWasmLocated = ""
```
[the remark this answers is not in the document]
Comment (vkubiv) [reply cmd/wallet-sdk-js/docs/usage.md:13]: It is a path to the folder where wasm located. WalletsWASMPath reads like full path to wasm, including wasm file name.

[the remark this answers is not in the document]
Comment (vkubiv) [reply cmd/wallet-sdk-js/docs/usage.md:13]: so it can be, `walletWASMLocationPath`

[the remark this answers is not in the document]
Comment (rolsonquadras) [reply cmd/wallet-sdk-js/docs/usage.md:13]: Do we have documentation on how the consumer can get the WASM file from the container registry? In other words, what the consumer of the library needs to do apart from adding the npm package to their package.json file ? @vkubiv @birtony

**Pull request merged**